staging-service/src/main/java/com/github/cafdataprocessing/services/staging/StagingController.java

```diff
@@ -179,10 +179,10 @@ public ResponseEntity<BatchList> getBatches(
             batchList.entries(batchFiles);
             return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(batchList);
         } catch (final InvalidTenantIdException ex) {
-            LOGGER.error("Invalid X-TENANT-ID ", ex);
+            LOGGER.warn("Invalid X-TENANT-ID {}", X_TENANT_ID);
             throw new WebMvcHandledRuntimeException(HttpStatus.BAD_REQUEST, ex.getMessage());
         } catch (final InvalidBatchIdException ex) {
-            LOGGER.error("Invalid batchId ", ex);
+            LOGGER.warn("Invalid batchId {}", ex.getMessage());
```

Comment:
Above I see we supply batchId in what looks like a similar case?
![image](https://github.com/CAFDataProcessing/staging-service/assets/18459684/4aa8fc8f-21fd-491b-bc1d-212963fa5904)

Reply:
Yes changed to `from`.